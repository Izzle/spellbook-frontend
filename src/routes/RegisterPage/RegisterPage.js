import React, { Component } from 'react';

export default class RegisterPage extends Component {
  render() {
    return (
      <section className='RegisterPage'>
          <h2>Register / Created an account here!</h2>
      </section>
    )
  }
}