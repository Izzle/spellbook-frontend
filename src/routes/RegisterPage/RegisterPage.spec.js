import React from 'react';
import renderer from 'react-test-renderer';
import ReactDOM from 'react-dom';
import RegisterPage from './RegisterPage';
import { BrowserRouter } from 'react-router-dom';


describe('<RegisterPage />', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<BrowserRouter><RegisterPage /></BrowserRouter>, div);
        ReactDOM.unmountComponentAtNode(div);
    });
    it('renders this UI as expected', () => {
        const tree = renderer.create(<BrowserRouter><RegisterPage /></BrowserRouter>).toJSON();
        expect(tree).toMatchSnapshot();
    });    
});