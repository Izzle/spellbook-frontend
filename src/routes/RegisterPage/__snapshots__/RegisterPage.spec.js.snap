// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RegisterPage /> renders this UI as expected 1`] = `
<section
  className="RegisterPage"
>
  <h2>
    Register / Created an account here!
  </h2>
</section>
`;
